src/playground/src/ui/screen/text.component.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { View } from 'react-native';
+import { NavigationScreenProps } from 'react-navigation';
+import {
+  withStyles,
+  ThemeType,
+  ThemedComponentProps,
+} from '@kitten/theme';
+import { Text as TextComponent } from '@kitten/ui';
+
+type Props = & ThemedComponentProps & NavigationScreenProps;
+
+class Text extends React.Component<Props> {
```

Comment:
I would rename screens to `%ControlName%Screen` so you don't need to make alias each time :)